install_scripts/templates/replicated-2.0.sh

```diff
@@ -272,8 +272,9 @@ build_replicated_opts() {
 
     if [ -n "$REPLICATED_OPTS" ]; then
         REPLICATED_OPTS=$(echo "$REPLICATED_OPTS" | sed -e 's/-e[[:blank:]]*HTTP_PROXY=[^[:blank:]]*//g')
+        REPLICATED_OPTS=$(echo "$REPLICATED_OPTS" | sed -e 's/-e[[:blank:]]*HTTPS_PROXY=[^[:blank:]]*//g')
```

Comment:
Separate line here instead of matching something like "HTTP|HTTPS" because not all sed versions support extended syntax